src/routes/balances/entities/balance.entity.ts

```diff
@@ -1,8 +1,14 @@
+import { Balance as DomainBalance } from '../entities/balance.entity';
+import { ApiProperty } from '@nestjs/swagger';
 import { TokenInfo } from './token-info.entity';
 
-export interface Balance {
-  tokenInfo: TokenInfo;
+export class Balance implements DomainBalance {
```

Comment:
`DomainBalance` could be confusing because it's not really a domain entity.

Reply:
What do you suggest?

Reply:
I would like use `domainBalance` for domain entities and `balance` for route entities.

Reply:
I don't get this, it should be importing/implementing the domain entity interface, right? It seems you're importing the same file (`'../entities/balance.entity'`)? 

Having that fixed, I think the naming it's ok. 🙂

The rest of the files have the same mistake I think.

Reply:
Yes you are right. I've fixed the imports either by removing them or by pointing them to the correct place. [cf549a9](https://github.com/5afe/safe-client-gateway-nest/pull/119/commits/cf549a953d7a617e18c8004937501fa7eac2edb7)